tests/Fixer/PhpUnit/PhpUnitOrderedCoversFixerTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitOrderedCoversFixer
+ */
+final class PhpUnitOrderedCoversFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return [
+            'skip on 1 or 0 occurrences' => [
+                '<?php
+                    class FooTest extends \PHPUnit_Framework_TestCase {
+                        /**
+                         * @covers Foo
+                         * @params bool $bool
+                         * @return void
+                         */
+                        public function testMe() {}
+
+                        /**
+                         * @params bool $bool
+                         * @return void
+                         */
+                        public function testMe2() {}
+                    }
+                ',
+            ],
+            'base case' => [
+                '<?php
+                    /**
+                     * @covers Bar
+                     * @covers Foo
+                     */
+                    class FooTest extends \PHPUnit_Framework_TestCase {}
+                ',
+                '<?php
+                    /**
+                     * @covers Foo
+                     * @covers Bar
+                     */
+                    class FooTest extends \PHPUnit_Framework_TestCase {}
+                ',
+            ],
+            'preserve positions if other docblock parts are present' => [
+                '<?php
+                    /**
+                     * @covers Bar
```

Comment:
also the case with
```
@covers Foo
@dataProvider provide
@covers Bar
```
would be nice to add (yes, most probably it works, I just want to have it to ensure it won't stop working after few months ;) )

Reply:
Done.